### PR TITLE
Net: removed non-functional DNS seeds

### PR DESCRIPTION
**Pull request by markasoftware**
Removed DNS seeds that do not return a list of Namecoin peers.

### CODE REVIEW

**Comment by JeremyRand** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

Mark Polyakov:
> Removed DNS seeds that do not return a list of Namecoin peers.

I've just sent Marius a reminder that his DNS seed is down, hopefully
he gets it fixed and we won't need to remove it.


-----BEGIN PGP SIGNATURE-----

iQIcBAEBCgAGBQJZYD3eAAoJELPy0WV4bWVwD88QALD+jXAiGLYMBgBF5E0rUKpU
BxGoMkd3h0civueMu5wLkMTPutwzvEeHmLSaw0f+385XJIaU1XyBIild4V8dOv8u
2VbC7RMsN+i0qNxR7NgxvHcLpEYIwvevXqUOPNt364eDCmjCAQvdwHK5JxDVDqs9
lmBnZR/jcrNSOY5VJFBDus9QpLf4sebVNrrIudNAFAJy6OLa2BO5fBi7YF0G7Zhi
I8+3nSG08vEZYupCcjhETaOswwaNsFitdwCxDAbXdQoYQUxu7vaM89yvmTTUQv3s
L17hxEYScHVyMsuedpWy5zej/uO5mkXODyAch9sBqyS1wZtkVDBUq3300yarn1kL
QNSdQZLjV8IkEv1nhKZsgZCS09sYP8T/bk3b7u2lNvFtiKyO6UbOpcOaNODv8gRd
5Tw2N9kX31kzv1y9eZtTAIIIyF5eBEOdw4J1vgzVD7RchBc/Pfe1lLo15z73PVI4
M8Gn4GuH67Tmos5j3FHw6YlNhlgTT8DZfL+IKQzu+n0Vjhfk/DYr1qqaAJZTI7e6
gCBXGGUC78+iXQ/VF2XrlPCHzOSVSBDd2vXnu1AWhGiTli53cdkbP0hJzLSrCkKT
8jwj+EopNabIzPPWARWvUxF904t/uYw5h2Tvv899ohR8Jtw1oyyH/H8DIL5CjoZV
5O3n2YtduCKPnEgHlGbc
=qQJ4
-----END PGP SIGNATURE-----

**Comment by JeremyRand** [comment]
I've just contacted George about his DNS seeds as well.

**Comment by mruddy** [comment]
Status as of this morning:

`namecoindnsseed.digi-masters.com`
* returns two addresses that both time out when attempting to TCP connect to port 8334:
  * 52.73.71.92
  * 52.71.185.125

`digi-masters.uk` and thus `namecoindnsseed.digi-masters.uk`
* name is not currently registered and could be registered by anyone at anytime

`dnsseed.namecoin.webbtc.com`
* server failure

`nmc.seed.quisquis.de`
* returns a DNS response over UDP that has the truncated flag set, causing resolution via TCP
* returns 31 addresses
  * 95.79.102.209
  * 93.123.80.47
  * 112.74.33.245
  * 188.165.254.191
  * 35.164.208.167
  * 139.129.167.217
  * 119.9.116.68
  * 176.31.128.16
  * 123.56.9.169
  * 188.65.212.138
  * 68.44.123.163
  * 112.74.171.220
  * 123.56.103.142
  * 144.76.13.207
  * 68.134.142.121
  * 24.59.208.4
  * 50.89.84.170
  * 158.129.212.251
  * 123.57.163.107
  * 68.168.116.154
  * 94.23.215.174
  * 222.106.12.31
  * 45.79.179.9
  * 37.191.155.182
  * 94.198.159.7
  * 198.15.127.242
  * 91.121.95.32
  * 173.230.146.227
  * 185.83.219.64
  * 74.84.128.158
  * 101.201.42.49

Testing all 33 seeds:
```
while read line; do nc -v -z -w 3 "$line" 8334; done < seeds.txt
nc: connect to 52.73.71.92 port 8334 (tcp) timed out: Operation now in progress
nc: connect to 52.71.185.125 port 8334 (tcp) timed out: Operation now in progress
Connection to 95.79.102.209 8334 port [tcp/*] succeeded!
nc: connect to 93.123.80.47 port 8334 (tcp) failed: No route to host
Connection to 112.74.33.245 8334 port [tcp/*] succeeded!
Connection to 188.165.254.191 8334 port [tcp/*] succeeded!
Connection to 35.164.208.167 8334 port [tcp/*] succeeded!
Connection to 139.129.167.217 8334 port [tcp/*] succeeded!
Connection to 119.9.116.68 8334 port [tcp/*] succeeded!
Connection to 176.31.128.16 8334 port [tcp/*] succeeded!
Connection to 123.56.9.169 8334 port [tcp/*] succeeded!
nc: connect to 188.65.212.138 port 8334 (tcp) failed: Connection refused
nc: connect to 68.44.123.163 port 8334 (tcp) timed out: Operation now in progress
Connection to 112.74.171.220 8334 port [tcp/*] succeeded!
nc: connect to 123.56.103.142 port 8334 (tcp) timed out: Operation now in progress
Connection to 144.76.13.207 8334 port [tcp/*] succeeded!
nc: connect to 68.134.142.121 port 8334 (tcp) timed out: Operation now in progress
nc: connect to 24.59.208.4 port 8334 (tcp) timed out: Operation now in progress
nc: connect to 50.89.84.170 port 8334 (tcp) timed out: Operation now in progress
Connection to 158.129.212.251 8334 port [tcp/*] succeeded!
nc: connect to 123.57.163.107 port 8334 (tcp) timed out: Operation now in progress
Connection to 68.168.116.154 8334 port [tcp/*] succeeded!
Connection to 94.23.215.174 8334 port [tcp/*] succeeded!
nc: connect to 222.106.12.31 port 8334 (tcp) timed out: Operation now in progress
Connection to 45.79.179.9 8334 port [tcp/*] succeeded!
Connection to 37.191.155.182 8334 port [tcp/*] succeeded!
Connection to 94.198.159.7 8334 port [tcp/*] succeeded!
Connection to 198.15.127.242 8334 port [tcp/*] succeeded!
Connection to 91.121.95.32 8334 port [tcp/*] succeeded!
Connection to 173.230.146.227 8334 port [tcp/*] succeeded!
nc: connect to 185.83.219.64 port 8334 (tcp) failed: Connection refused
Connection to 74.84.128.158 8334 port [tcp/*] succeeded!
nc: connect to 101.201.42.49 port 8334 (tcp) failed: Connection refused
```

From node's debug.log:
```
2017-11-22 12:28:28 Loaded 906 addresses from peers.dat  7ms
2017-11-22 12:28:39 Loading addresses from DNS seeds (could take a while)
2017-11-22 12:28:52 33 addresses found from DNS seeds
2017-11-22 12:28:52 dnsseed thread exit
```

Node does not connect to any of the addresses that it finds.

This does work in the console: `addnode 95.79.102.209 onetry`

**Comment by JeremyRand** [comment]
George seems to be unreachable via email and Twitter DM regarding his broken seeds, so Concept ACK on removing them.  If he re-establishes contact, we can discuss re-adding them then.

**Comment by mruddy** [comment]
tested ACK

**Comment by JeremyRand** [comment]
Needs rebase.

**Comment by markasoftware** [comment]
should be ok now

File: src/chainparams.cpp [hunk 1]
```diff
@@ -161,12 +161,9 @@ class CMainParams : public CChainParams {
         assert(genesis.hashMerkleRoot == uint256S("0x41c62dbd9068c89a449525e3cd5ac61b20ece28c3c38b3f35b2161f0e6d3cb0d"));
 
         // Note that of those with the service bits flag, most only support a subset of possible options
-        vSeeds.emplace_back("namecoindnsseed.digi-masters.com", false);
-        vSeeds.emplace_back("namecoindnsseed.digi-masters.uk", false);
         vSeeds.emplace_back("nmc.seed.quisquis.de", false);
-        vSeeds.emplace_back("dnsseed.namecoin.webbtc.com", false);
         vSeeds.emplace_back("seed.nmc.markasoftware.com", false);
-
+
```
Comment (JeremyRand) [inline src/chainparams.cpp]: Can you please remove the whitespace diff on this line?

**Comment by JeremyRand** [comment]
(Triggering a rebuild on Travis.)

**Comment by JeremyRand** [comment]
utACK 15778451373231e0d7b42d3fc68bf200f689a49c

**Pull request merged**